google-cloud-bigtable/lib/google/cloud/bigtable/column_family_map.rb

```diff
@@ -160,9 +167,14 @@ def add name, gc_rule: nil
         #     column_families.update "cf2", gc_rule: rule_union
         #   end
         #
-        def update name, gc_rule: nil
+        def update name, positional_gc_rule = nil, gc_rule: nil
```

Comment:
We don't need the positional `gc_rule` on `update`, afaik, just on `add`, for `create_table` blocks.

Reply:
This is true. I'll remove.

So it looks like the current [`ColumnFamilyMap`](https://github.com/googleapis/google-cloud-ruby/blob/google-cloud-bigtable/v0.6.2/google-cloud-bigtable/lib/google/cloud/bigtable/table/column_family_map.rb) only has `#add` and `#remove`. So raising when adding a name that already exists is likely another breaking change. I think its worth it however. WDYT?